.ci_support/linux_ppc64le_cuda_compiler_version11.2.yaml

```
@@ -0,0 +1,26 @@
+c_compiler:
+- gcc
+c_compiler_version:
+- '10'
+cdt_name:
+- cos7
+channel_sources:
+- conda-forge
+channel_targets:
+- conda-forge main
+cuda_compiler_version:
+- '11.2'
```

Not sure why this winds up missing here, but that is part of the issue. Other files here might be missing this as well.

Since this is an autogenerated file, it means we are missing something in the migrator (maybe just these same 2 lines?).

```suggestion
cuda_compiler:
 - nvcc
cuda_compiler_version:
- '11.2'
```

---

Good point... let's give it a try here, if it works I'll send a patch to the pinning feedstock and get it fixed there.

---

Sounds good. It looked like another variant file may have needed this as well, but just commented on this one for simplicity

Should add as mentioned in issue ( https://github.com/conda-forge/conda-forge-pinning-feedstock/issues/2679 ), we may run into this Docker image issue ( https://github.com/conda-forge/docker-images/issues/208 ). Open to other thoughts there if anyone has any 🙂

---

Yeah looks like it works... but not if I add it to the migrator and rerender (which was what I did), so it means we still need to find out why `cuda_compiler` is missing in the autogen'd CI files...

---

See [8d203f6](https://github.com/conda-forge/cub-feedstock/pull/17/commits/8d203f67219264eb4a04c43219cab654596eee4e). The rerender would kill it.

---

Yeah IIRC re-rendering pulls in the latest migrator from conda-forge-pinning. So it is not surprising that got overwritten. Think there is a flag `conda-smithy` will take to disable this behavior. Think if we update the migrator we should be good to go on this front.

---

@jakirkham Sounds like I should send a patch to CF-pinning to fix the migrator first?

---

See https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/2687.

---

@jakirkham looks like https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/2687 didn't fix it...
```
INFO:conda_smithy.configure_feedstock:Re-rendered with conda-build 3.21.8, conda-smithy 3.19.0, and conda-forge-pinning 2022.03.31.18.34.20
[299](https://github.com/conda-forge/cub-feedstock/runs/5776675647?check_suite_focus=true#step:3:299)
INFO:conda_smithy.configure_feedstock:No changes made. This feedstock is up-to-date.
```

---

but I thought `2022.03.31.18.34.20` was used in rerendering? (see the log above)